dspace-spring-rest/src/main/java/org/dspace/app/rest/repository/CommunityRestRepository.java

```diff
@@ -165,4 +196,46 @@ public CommunityResource wrapResource(CommunityRest community, String... rels) {
         return new CommunityResource(community, utils, rels);
     }
 
+    @Override
+    @PreAuthorize("hasAuthority('ADMIN')")
```

Comment:
Should we check for WRITE on the community instead than general admin rights?

Reply:
@abollini is correct. this annotation could be updated to
@PreAuthorize("hasPermission(#id, 'COMMUNITY', 'WRITE')")